apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/KeyStoreController.java

```diff
@@ -70,12 +75,41 @@ public KeyStoreController(KMSService KMSService, SecureStorageService secureStor
         this.KMSService = KMSService;
         this.secureStorageService = secureStorageService;
         this.maxKeyStoreSize = maxKeyStoreSize;
+        ensureTempUploadDirOnInit();
```

Comment:
I would suggest moving this invocation to the static block of the class and 
remove the condition "if (MULTI_PART_CONFIG == null)" from ensureTempUploadDirOnInit method;

Reply:
fixed

> I would suggest moving this invocation to the static block of the class and remove the condition "if (MULTI_PART_CONFIG == null)" from ensureTempUploadDirOnInit method;

fixed